src/azsdkdemoprocessor/pom.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>demo.sdk.az</groupId>
+  <artifactId>processor</artifactId>
```

Comment:
The group and artifactIds are the same as the previous pom. If both are required, they should have different artifactId.

Reply:
I'll removed the other one.